loop/views.py

```diff
@@ -77,7 +77,7 @@ def login(request):
                     'phone_digits':loop_user[0].village.block.district.state.phone_digit,
                     'phone_start':loop_user[0].village.block.district.state.phone_start,
                     'preferred_language':loop_user[0].preferred_language.notation,
-                    'country':loop_user[0].village.block.district.state.country.country_name}))
+                    'country':loop_user[0].village.block.district.state.country.country_name,'role':loop_user[0].role,'farmer_phone_mandatory':loop_user[0].farmer_phone_mandatory}))
```

Comment:
expecting 'State' in response at client, but not sending it from server. Either remove State from client or add State in response.